pkg/util/sys/device.go

```diff
@@ -258,7 +258,7 @@ func GetDeviceFilesystems(device string, executor exec.Executor) (string, error)
 
 // GetDiskUUID look up the UUID for a disk.
 func GetDiskUUID(device string, executor exec.Executor) (string, error) {
-	if _, err := os.Stat("/usr/sbin/sgdisk"); err != nil {
+	if _, err := osexec.LookPath(sgdiskCmd); err != nil {
```

Comment:
Shouldn't the return of the `LookPath()` function be used then in the function?

Reply:
It's not necessary because executor.ExecuteCommandWithOutput() internally calls exec.LookPath(). In addition, if we use the return value of cmd, it complicates the code like the following.

```
...
var cmdPath // This lline and the next line should exist to reuse `cmdPath` after `if` statement.
var err error
if cmdPath, err = osexec.LookPath(sgdiskCmd); err != nil {
  ...
}
...
```

Reply:
If the executor does a lookup why do we need to check the existence of the binary? This could be an error returned by the executor right?

Reply:
Sounds good!